Automattic/sensei
# Add Sensei Home Structure

Fixes #5625

### Changes proposed in this Pull Request

* Move almost all of the stuff related to the Extensions page to the new page, Home;
* Implement "metaboxes"-like blocks called sections, using postbox CSS classes;
* Split Home page into many smaller sections, as a placeholder;
* Start adapting styles for notices and for the Sensei Pro ad;

### Testing instructions

On a WP installation with this version of the plugin being used, on the admin panel, click on the Sensei LMS -> Home (should be at the top of the menu) and verify if the page "works" correctly and looks somewhat similar to the latest design iteration;

### To do

There are a bunch of tasks still left to do, like:

1. Add a filter so Sensei Pro (or any other component, really) can inject content on the top of the page;
2. Adjust styling of the extensions block (maybe part of #5630?);
3. Remove most of the unused CSS related to the navbar and old code from the old extensions page;
4. Probably fix the badge counter on the menu so it doesn't count outdated plugins;
5. Finish implementing JITMs using WP notices (maybe part of #5674?);
6. Probably convert extensions loading code to report errors as WP notices;
